inginious/frontend/installer.py

```diff
@@ -569,7 +569,7 @@ def configure_authentication(self, database):
                                    "password": UserManager.hash_password(password),
                                    "bindings": {},
                                    "language": "en",
-                                   "timezone": "UTC"})
+                                   "timezone": "None"})
```

Comment:
Replaced "UTC" by another default string value to be sure it is well understood no timezone is selected.

We could replace with None value but it would be passed to html as a variable name and would need to be transformed into  a string. It would be a bit more messy for the code. What's best ? Leave a string to represent no selected timezone ? Something else ?